oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/requests/search/aggregation/TermsAggregation.java

```diff
@@ -58,6 +58,8 @@ public void serialize(final TermsAggregation value, final JsonGenerator gen,
                     if (value.getOrder() != null) {
                         writeOrder(value, gen);
                     }
+                    gen.writeStringField("execution_hint", "map");
+                    gen.writeStringField("collect_mode", "breadth_first");
```

Comment:
Please don't do this in the codec level.

- Add fields `executionHint`, `collectMode` in `TermsAggregation` (you can make them enum type considering their possible values are limited in ES's spec).
- Add fields `executionHint`, `collectMode` in `TermsAggregationBuilder` and corresponding builder methods, pass the two to `TermsAggregation` constructor in `build` method.
- Write the 2 fields here only when they are not empty/blank.

Then you should change set the 2 fields in topology query.